docs/docs/submit-to-starter-library.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Submit to Starter Library
+---
+
+Have you created a Gatsby starter you'd like to add to the [Starter Library](/starters/)? Follow these instructions.
+
+## Steps
+
+There are only two major steps :)
+
+1.  If this is your first contribution to the Gatsby open source repo, follow the [Contribution guidelines](/docs/how-to-contribute/#contributing-to-the-repo).
+
+2.  Edit the [`starters.yml`](https://github.com/gatsbyjs/gatsby/blob/master/docs/starters.yml) file by adding your starter information to the bottom of the list in the following format:
+
+```yaml
+- url: Link to a demo of your starter
+  repo: Link to GitHub repo
+  description: Your starter description
+  # These correspond to the category filters in the library
+  # Make an effort to use the existing tags, and add more if needed!
+  tags:
+    - Redux
+  # Add your site features
+  # These will be included on your starter's detail page.
+  features:
+    - Blog post listing with previews (image + summary) for each blog post
+    - Categories and tags for blog posts with pagination
+```
+
+Check out the `starters.yml` file for examples.
```

Comment:
Should we link this?

Reply:
Might as well. I didn't because it's linked above (I just did it as an extra supporting note) but it sure doesn't hurt!